enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,239 @@
+  Windows nodes.
+* Create a Service Monitor in `openshift-windows-machine-config-operator`
+  namespace for Service created above.
+  
+To display node graphs WMCO has done the following:
+
+* Add custom Prometheus rules in `openshift-windows-machine-config-operator`
+  namespace. The custom recording rules are created using Windows metrics
+  exposed by the windows_exporter and have the same names as Linux
+  recording rules. This is to make use of same console queries as Linux.
+* Note that WMCO is unable to display pod graphs for the Windows Nodes
+  with the current implementation. See [Risks and Mitigations](#risks-and-mitigations)
+  for details.
+  
+### Future Plans
+
+#### Displaying Console Graphs
+
+* As we move forward, our plan to display monitoring graphs is to create a
+ [common interface](https://issues.redhat.com/browse/WINC-530) for Windows
+  and Linux recording rules. Monitoring team will define recording rules for the
+  metrics that have different `metric labels` for Linux and Windows. The
+  differences in `metric labels` for metrics used for Node graphs and pod graphs
+  are displayed in the tables below.
+  The Windows team will align the Windows recording rules with these new
+  recording rules. The recording rules for Windows will be managed by
+  WMCO. This set of common recording rules for monitoring will return results
+  for both Linux and Windows nodes for a single query.The console queries
+  currently use some raw metrics such as `node_filesystem_size_bytes`,
+  `node_filesystem_free_bytes` etc. They would need to be updated to include
+  the new recording rules in place of using raw metrics. This will ensure that
+  we have a consistent user experience for monitoring across Linux and Windows.
+* In the cases where `metric labels` are equivalent, we plan to relabel the
+  Windows metrics to align with the Linux metrics.
+
+##### Node Metrics
+
+| Node Exporter                  | Windows Exporter                 | Label Difference                                                         |
+|--------------------------------|----------------------------------|--------------------------------------------------------------------------|
+| node_memory_MemTotal_bytes     | windows_cs_physical_memory_bytes | -                                                                        |  
+| node_memory_MemAvailable_bytes | windows_memory_available_bytes   | -                                                                        |  
+| node_filesystem_size_bytes     | windows_logical_disk_size_bytes  | Missing Labels: (device, mountpoint, fstype) Additional label : (volume) |
+| node_filesystem_free_bytes     | windows_logical_disk_free_bytes  | Missing Label: device, mountpoint, fstype) Additional label : (volume)   |
+| node_cpu_seconds_total         | windows_cpu_time_total           | Missing Label : cpu Additional Label: core                               |
+
+##### Pod Metrics
```

Comment:
same as above, we need to identify where recording rules would need to be created/updated vs. where WMCO can modify the metrics to look like metrics from Linux nodes.